torch/csrc/utils/schema_info.cpp

```diff
@@ -59,6 +59,39 @@ bool SchemaInfo::is_mutable(c10::string_view name) {
   return is_mutable(*index);
 }
 
+bool SchemaInfo::is_non_deterministic() const {
```

Comment:
add a comment that this came from JIT (note: it probably makes sense to put the comment in the cpp file cause that's where the implementation is: i.e., I wouldn't wonder where this came from just by looking at the function signature in the .h file, but in the .cpp file I might see this list of ops and wonder where it came from)

Reply:
nit: i'd probably name it `is_nondeterministic` for consistency, unless there's a reason to prefer `non_deterministic`